go/host/p2p/p2p.go

```diff
@@ -70,8 +70,9 @@ type p2pImpl struct {
 	p2pTimeout        time.Duration
 	logger            gethlog.Logger
 	peerTracker       *peerTracker
-	hostGauges        map[string]map[string]gethmetrics.Gauge
-	metricsRegistry   gethmetrics.Registry
+	// hostGauges holds a map of gauges per event per host to track p2p metrics and health status
```

Comment:
it's the other way round?

per host per event

Reply:
```suggestion
	// hostGauges holds a map of gauges per host per event to track p2p metrics and health status
```

yep!